sui/src/rest_server.rs

```diff
@@ -166,32 +166,27 @@ Generate OpenAPI documentation.
     path = "/docs",
     tags = [ "docs" ],
 }]
-async fn docs(
-    rqctx: Arc<RequestContext<ServerContext>>,
-) -> Result<HttpResponseOk<DocumentationResponse>, HttpError> {
+async fn docs(rqctx: Arc<RequestContext<ServerContext>>) -> Result<Response<Body>, HttpError> {
```

Comment:
Why are we changing this function?

Reply:
Adding CORS headers to endpoint to be consistent with all other endpoints.